data/private

```diff
@@ -126,22 +126,24 @@ full:instant.arubanetworks.com
 full:setmeup.arubanetworks.com
 
 # ASUS router
-full:router.asus.com
+router.asus.com
+asusrouter.com
```

Comment:
Use the `full:` prefix whenever you can to avoid mismatch.

Reply:
Using `full:` can't match `www.asusrouter.com`, so just use `domain: asusrouter.com` for all situations.

Reply:
> It doesn't matter if the url has a `www` prefix or not, both can be accessed. Using `full:` can't match `www.asusrouter.com`, so just use `domain:asusrouter.com` for all situations, and this domain only used for router management.

Yes, it does matter. This is the `private` category and currently we are abusing it.
So I want only the necessary domains to be included in this category, not those we can not determine whether it is used for intranet access or not.

Reply:
> It doesn't matter if the url has a `www` prefix or not, both can be accessed. Using `full:` can't match `www.asusrouter.com`, so just use `domain:asusrouter.com` for all situations, and this domain only used for router management.

Yes, it does matter. This is the `private` category and currently we are abusing it.
So I want only the necessary domains to be included in this category, not those we can not determine are used for intranet access.

Reply:
> It doesn't matter if the url has a `www` prefix or not, both can be accessed. Using `full:` can't match `www.asusrouter.com`, so just use `domain:asusrouter.com` for all situations, and this domain only used for router management.

Yes, it does matter. This is the `private` category and currently we are abusing it.
So I want only the necessary domains to be included in this category, not those we can not determine are used for intranet access.

Reply:
Added `full:asusrouter.com` and `full:www.asusrouter.com`.